lib/rubocop/cop/lint/detect_invalid_pack_directives.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      class DetectInvalidPackDirectives < RuboCop::Cop::Base
```

Comment:
Can we add docs to the class here, similar to what Rubocop does upstream - https://github.com/rubocop/rubocop/blob/24f6e997b9f30e90eb964248c674417d0c0f7e3a/lib/rubocop/cop/lint/non_atomic_file_operation.rb#L6-L43 or https://github.com/rubocop/rubocop/blob/24f6e997b9f30e90eb964248c674417d0c0f7e3a/lib/rubocop/cop/lint/ambiguous_operator_precedence.rb#L6-L29 etc